libraries/AP_GPS/AP_GPS.h

```diff
@@ -242,6 +283,14 @@ class AP_GPS
         float relPosD;                     ///< Reported Vertical distance in meters
         float accHeading;                  ///< Reported Heading Accuracy in degrees
         uint32_t relposheading_ts;        ///< True if new data has been received since last time it was false
+
+        // Integrity message information 
```

Comment:
These should be gated with the define suggested earlier.

This PR should result in no compiler output change on boards which don't support the sending of this message.